docs/tutorials/grover_with_sampler.ipynb

```diff
@@ -7,17 +7,17 @@
    "source": [
```

Comment:
<p>This part reminds me of when I added the explanation about <code>QuantumCircuit.draw('mpl')</code>.Should we consider that an option? I think it's more informative (without deviating too much) this way and therefore better.  </p><p> </p><p>However, one could also argue that this extra explanation is more fundamental to understand how to use the primitive as it doesn't make sense not to explain how to choose a backend, for example, even if there are default options. Similarly, I believe creating a service instance is a fundamental part of the Runtime workflow, so putting some attention into it is good. (the original version creates a <code>service</code> variable that is not used at all )  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I'd add that explanation about the <code>service</code> instance to make sure the user has a full understanding of how the <code>Sampler</code> works. What do you think?</p><p> </p><p>The <code>backend</code> part I'm fine with.  </p>